scripts/src/test/org/oppia/android/scripts/RegexPatternValidationCheckTest.kt

```diff
@@ -0,0 +1,136 @@
+package org.oppia.android.scripts
+
+import java.io.File
+import org.oppia.android.scripts.RegexPatternValidationCheck
+import org.junit.Test
+import org.junit.Rule
+import org.junit.Before
+import org.junit.After
+import org.junit.rules.TemporaryFolder
+import org.oppia.android.scripts.ScriptResultConstants
+import com.google.common.truth.Truth.assertThat
+import org.oppia.android.testing.assertThrows
+import java.io.PrintStream
+import java.io.ByteArrayOutputStream
+
+class RegexPatternValidationCheckTest {
+
+  @Rule
+  @JvmField
+  public var tempFolder: TemporaryFolder = TemporaryFolder()
+
+  @Before
+  fun initTestFilesDirectory() {
+    tempFolder.newFolder("testfiles")
+  }
+
+  private val outContent: ByteArrayOutputStream = ByteArrayOutputStream()
+  private val originalOut: PrintStream = java.lang.System.out
```

Comment:
Keep these at the top before putting any functions 
so format of test file will be like:

1. properties
2. functions

Reply:
done